tests/data/esc_sql.php

```diff
@@ -9,13 +9,35 @@
 use function PHPStan\Testing\assertType;
 
 // Indexed array as parameter
-assertType('array<int, string>', esc_sql(['someValue', 'toEscape']));
+assertType("array{string, string}", esc_sql(['someValue', 'toEscape']));
```

Comment:
```suggestion
assertType('array{string, string}', esc_sql(['someValue', 'toEscape']));
```

Reply:
right.. I had the wrong type there initially and forgot to remove them. removed all in a dedicated commit